docs/admin/machine.rst

```diff
@@ -303,6 +303,27 @@ To enable this service, add ``weblate.machinery.youdao.YoudaoTranslation`` to
     :setting:`MT_YOUDAO_SECRET`
     `Youdao Zhiyun Natural Language Translation Service <https://ai.youdao.com/product-fanyi.s>`_
 
+.. _netease-translate:
+
+Netease Sight API machine translation
+-------------------------------------
+
+.. versionadded:: 3.2
```

Comment:
It won't make it into 3.2, sorry.

Reply:
So which version should be here?

Reply:
Next version will be 3.3, 3.2 was released today.